src/services/stripeService.js

```diff
@@ -0,0 +1,37 @@
+export const getAllProducts = () =>
+  fetch(`/api/products-stripe`).then((res) => res.json());
+
+export const getAllPrices = () =>
+  fetch(`/api/products-stripe/price`).then((res) => res.json());
+
+export const getSubscriptions = async ({ jwt }) => {
+  const res = await fetch(`/api/users/subscription`, {
+    headers: {
+      Authorization: "Bearer " + jwt,
+    },
+  });
+
+  const body = await res.json();
+
+  if (!res.ok) {
+    throw body;
+  }
+
+  return body;
+};
+
+export const createCheckoutSession = ({ priceId, jwt }) =>
+  fetch(`/api/products-stripe/price/${priceId}/create-checkout-session`, {
```

Comment:
https://stackoverflow.com/questions/44820568/set-default-header-for-every-fetch-request